mysql-test/suite/innodb/t/percona_alter_debug.test

```diff
@@ -0,0 +1,78 @@
+# Tests in this file use both debug-only facilities as well as debug sync point.
```

Comment:
@dlenev in 8.4, please move percona-specific InnoDB tests to "percona_innodb" test suite.
Also, rename the test to just `alter_debug`.

Reply:
Done.